tests/integration/provider_gov_hooks.go

```diff
@@ -98,11 +98,17 @@ func (s *CCVTestSuite) TestGetConsumerAdditionLegacyPropFromProp() {
 			name:                    "msgs contain an invalid legacy prop",
 			propMsg:                 &v1.MsgExecLegacyContent{},
 			expectConsumerPropFound: false,
-			expPanic:                true,
+			expPanic:                false,
+		},
+		{
+			name:                    "msg contains a prop of legacy ConsumerAdditionProposal type - hook should NOT create a new proposed chain",
+			propMsg:                 addConsumerPropLegacy,
+			expectConsumerPropFound: false,
+			expPanic:                false,
 		},
 		{
-			name:                    "msg contains a prop of ConsumerAdditionProposal type - hook should create a new proposed chain",
-			propMsg:                 addConsumerProp,
+			name:                    "msg contains a prop of legacy ConsumerAdditionProposal type - hook should create a new proposed chain",
```

Comment:
Not clear what this test case is doing differently from the name. They both state "msg contains a prop of legacy ConsumerAdditionProposal type" but the action of the hook is different.